### PR TITLE
Autotests: #5818 - tests for save PNG images to PNG/SVG files

### PR DESCRIPTION
**Pull request by ivanmaslow**
Added expected screenshots;

## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [ ] unit-tests written
- [x] e2e-tests written
- [ ] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"
- [x] reviewers are notified about the pull request

### CODE REVIEW

File: ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts [hunk 1]
```diff
@@ -1948,7 +1948,7 @@ test.describe('Image files', () => {
 
   test('Verify that added by Tool SVG images with elements are displayed on preview and can be saved together to PNG file with the correct positions and layers after selection, moving actions of images', async () => {
     /**
-     * Test case: https://github.com/epam/Indigo/issues/2161
+     * Test case: https://github.com/epam/Indigo/issues/2162
```
Comment (Zhirnoff) [inline ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts:1951]: Why did you change the task number?

File: ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts [hunk 2]
```diff
@@ -1974,7 +1974,7 @@ test.describe('Image files', () => {
 
   test('Verify that added by Tool SVG images with elements are displayed on preview and can be saved together to PNG file with the correct positions and layers after scaling actions of images', async () => {
     /**
-     * Test case: https://github.com/epam/Indigo/issues/2161
+     * Test case: https://github.com/epam/Indigo/issues/2162
```
Comment (Zhirnoff) [inline ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts:1977]: Why did you change the task number?

File: ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts [hunk 3]
```diff
@@ -2006,7 +2006,7 @@ test.describe('Image files', () => {
 
   test('Verify that added by Tool SVG images with elements are displayed on preview and can be saved together to PNG file with the correct positions and layers after deleting actions of images', async () => {
     /**
-     * Test case: https://github.com/epam/Indigo/issues/2161
+     * Test case: https://github.com/epam/Indigo/issues/2162
```
Comment (Zhirnoff) [inline ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts:2009]: Why did you change the task number?

File: ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts [hunk 4]
```diff
@@ -2033,7 +2033,7 @@ test.describe('Image files', () => {
 
   test('Verify that added by Tool SVG images with elements are displayed on preview and can be saved together to PNG file with the correct positions and layers after copying actions of images', async () => {
     /**
-     * Test case: https://github.com/epam/Indigo/issues/2161
+     * Test case: https://github.com/epam/Indigo/issues/2162
```
Comment (Zhirnoff) [inline ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts:2036]: Why did you change the task number?

**Review by Zhirnoff — APPROVED** [review]

File: ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts (hunk 4 above)
Comment (ivanmaslow) [reply ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts:2036]: Because the number of task isn't correct

File: ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts (hunk 3 above)
Comment (ivanmaslow) [reply ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts:2009]: Because the number of task isn't correct

File: ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts (hunk 2 above)
Comment (ivanmaslow) [reply ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts:1977]: Because the number of task isn't correct

File: ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts (hunk 1 above)
Comment (ivanmaslow) [reply ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts:1951]: Because the number of task isn't correct
